www/src/Contexts/AppContext.jsx

```diff
@@ -15,12 +15,17 @@ yup.addMethod(yup.string, 'validateColor', function () {
 	);
 });
 
+const isEnabledValue = (value) => {
```

Comment:
This will fail on arrays, [0,1] when it shouldn't. But you can flip it and it works fine, I'll take care of it and merge it in